README.md

```diff
@@ -23,7 +22,7 @@ is rather simple.
 
 Although significant changes have been done for the 2nd version
 (a complete rewrite) the API is compatible with the 1.0.0 release. Except for
-the namespace which is now `nlohmann::json_schema.
+the namespace which is now \`nlohmann::json_schema.
```

Comment:
Here the error is actually a missing back-tick after json-schema
```suggestion
the namespace which is now `nlohmann::json_schema`.
```

Reply:
Thanks for the review. Yeah it was misspelled back then